tools/cockpit.spec

```diff
@@ -155,8 +162,13 @@ echo '{ "linguas": null, "machine-limit": 5 }' > %{buildroot}%{_datadir}/%{name}
 echo '%dir %{_datadir}/%{name}/base1' > base.list
 find %{buildroot}%{_datadir}/%{name}/base1 -type f >> base.list
 
+%if %{defined build_dashboard}
 echo '%dir %{_datadir}/%{name}/dashboard' >> dashboard.list
 find %{buildroot}%{_datadir}/%{name}/dashboard -type f >> dashboard.list
+%else
+rm -rf %{buildroot}/%{_datadir}/%{name}/dashboard
+touch dashboard.list
```

Comment:
This else branch looks a bit awkward -- shouldn't we rather fix that in the upstream build system? I. e. not build/install the dashboard if `--disable-ssh` is given?